src/ui/common/rich-text-editor/RichTextEditor.js

```diff
@@ -0,0 +1,209 @@
+import React, { Component, createRef } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import ReactQuill, { Quill } from 'react-quill-2';
+import 'react-quill-2/dist/quill.snow.css';
+
+import EditorToolbar from 'ui/common/rich-text-editor/EditorToolbar';
+import LinkConfigModal from 'ui/common/modal/LinkConfigModal';
+import store from 'state/store';
+import { setVisibleModal } from 'state/modal/actions';
+import { getVisibleModal } from 'state/modal/selectors';
+
+const BlockEmbed = Quill.import('blots/block/embed');
+
+function DividerBlot(...args) {
+  return Reflect.construct(BlockEmbed, args, DividerBlot);
+}
+Object.setPrototypeOf(DividerBlot.prototype, BlockEmbed.prototype);
+Object.setPrototypeOf(DividerBlot, BlockEmbed);
+DividerBlot.blotName = 'divider';
+DividerBlot.tagName = 'hr';
+
+Quill.register(DividerBlot);
+
+const txtArea = document.createElement('textarea');
+txtArea.classList.add('html-editor');
+txtArea.style.display = 'none';
+
+function history(value) {
```

Comment:
please use arrow functions

Reply:
Those are quill handler functions and shouldn't be arrow functions because the `this` reference is required to access the quill instance.

Reply:
ok then, thanks